src/curses.h

```diff
@@ -15,6 +15,10 @@
   #undef KEY_EVENT
   #include <ncurses/ncurses.h>
 #else
-  #include <ncurses.h>
+  #ifdef __NetBSD__
```

Comment:
Is it possible to flatten this indentation?

```
#elif __NetBSD__
```

Reply:
Done.